src/scss/_packages.scss

```diff
@@ -2,80 +2,172 @@
 @import "mixins";
 
 .r-tiles {
-    @apply items-center;
-
     @screen md {
-        @apply flex;
+        @apply grid grid-cols-2 grid-flow-row-dense gap-8;
     }
 
-    .r-tile {
-        @apply rounded shadow border border-gray-300 cursor-pointer p-4 font-display;
+    .package {
 
-        width: 525px;
-        max-width: 525px;
+        .r-tile {
+            @apply rounded-xl shadow border border-gray-300 p-4 font-display;
+            @apply flex flex-col;
 
-        .logo {
-            @apply mr-4;
+            width: 100%;
 
-            width: 48px;
-            height: 48px
-        }
+            .r-tile-header {
+                @apply flex flex-row items-center mb-4;
+
+                .logo {
+                    max-width: 20%;
+                    height: 32px;
+                }
 
-        &.featured {
-            @apply cursor-default;
+                .header-link {
+                    @apply text-blue-600 rounded;
 
-            max-height: 350px;
-            height: 350px;
+                    padding: 8px 16px 8px 16px;
+
+                    &:hover {
+                        @apply text-blue-700;
+
+                        box-shadow: 0 4px 4px rgba(0, 0, 0, 0.2), 0 4px 4px rgba(0, 0, 0, 0.14), 0 1px 5px rgba(0, 0, 0, 0.12);
+                    }
 
-            .description {
-                @apply overflow-ellipsis overflow-hidden;
+                    // Because of the custom box-shadow rule, the tailwind helper for applying
+                    // a focus ring wasn't working well. That is, focus:ring-2 wasn't working.
+                    // That's why the ring rules are being applied manually to the :focus
+                    // pseudo-selector.
+                    &:focus {
+                        @apply ring-2 ring-blue-700;
+                    }
+                }
             }
 
-            .featured-ex-container {
-                @apply overflow-auto;
+            .r-tile-content {
+                @apply flex flex-col justify-evenly;
 
-                height: 64px;
-                max-height: 64px;
+                height: 100%;
 
-                ul {
-                    li {
-                        @apply text-sm truncate overflow-ellipsis;
+                .title-link {
+                    @apply text-lg;
 
-                        width: 300px;
+                    &:hover {
+                        @apply text-blue-700 underline;
                     }
                 }
-            }
-        }
 
-        @include transition;
+                pulumi-accordion {
+                    $svgDefaultStrokeColor: #000000;
```

Comment:
If this is the way we want accordions to look by default, we should add an entry for it in `scss/components` folder for it, rather than treat it individually here.

Reply:
> If this is the way we want accordions to look by default

I don't think we do? I believe it is specific to the packages view.